runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/CompositeParsers/DelegatorInvokesDelegateRuleWithArgs.stg

```diff
@@ -28,5 +28,5 @@ WS : (' '|'\n') -> skip ;
 
 slaveGrammar(grammarName) ::= <<
 parser grammar S;
-a[int x] returns [int y] : B {<write("\"S.a\"")> $y=1000;} ;
```

Comment:
that would not insert a ‘;’ between the 2 productions, hence would not work in python

> Le 21 juin 2015 à 03:36, Sam Harwell <notifications@github.com> a écrit :
> 
> In runtime-testsuite/resources/org/antlr/v4/test/runtime/templates/CompositeParsers/DelegatorInvokesDelegateRuleWithArgs.stg <https://github.com/antlr/antlr4/pull/927#discussion_r32887275>:
> 
> > @@ -28,5 +28,5 @@ WS : (' '|'\n') -> skip ;
> >  
> >  slaveGrammar(grammarName) ::= <<
> >  parser grammar S;
> > -a[int x] returns [int y] : B {<write("\"S.a\"")> $y=1000;} ;
> Interesting. I think we could actually use the following as a more "ANTLR-y" way to do this. How do you feel about this as a general pattern for executing a sequence of multiple actions at a specific location in a test?
> 
> {<write("\"S.a\"")>} {$y=1000;}
> —
> Reply to this email directly or view it on GitHub <https://github.com/antlr/antlr4/pull/927/files#r32887275>.
>